<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Yammys-MacBook-Pro.local" name="test_Apple iPad Pro 9.3.2" tests="2" failures="2" timestamp="19 Jul 2017 12:08:28 GMT" time="254.633" errors="0">
  <testcase name="@AfterClass afterClass" time="0.0" classname="com.pcloudy.testng.TestClass1">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.pcloudy.testng.TestSetUp.afterClass(TestSetUp.java:61)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- @AfterClass afterClass -->
  <testcase name="@AfterClass afterClass" time="0.0" classname="com.pcloudy.testng.TestClass2">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.pcloudy.testng.TestSetUp.afterClass(TestSetUp.java:61)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- @AfterClass afterClass -->
  <testcase name="test1" time="26.041" classname="com.pcloudy.testng.TestClass1">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.pcloudy.testng.TestClass1.test1(TestClass1.java:30)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
  <testcase name="test2" time="1.111" classname="com.pcloudy.testng.TestClass2">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.pcloudy.testng.TestClass2.test2(TestClass2.java:29)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
</testsuite> <!-- test_Apple iPad Pro 9.3.2 -->
